src/adapters/supabase/helpers/log.ts

```diff
@@ -27,11 +27,12 @@ export class GitHubLogger implements Logger {
   private app;
   private logEnvironment;
   private logQueue: Log[] = []; // Your log queue
-  private maxConcurrency = 5; // Maximum concurrent requests
-  private retryLimit = 3; // Maximum number of retries
+  private maxConcurrency = 6; // Maximum concurrent requests
   private retryDelay = 1000; // Delay between retries in milliseconds
   private throttleCount = 0;
 
+  private retryLimit = process.env.LOG_RETRY || 0; // Retries disabled by default
```

Comment:
We shouldn't read .env variable here. It should be read [here](https://github.com/devpanther/bounty-bot/blob/logs-to-supabase/src/bindings/config.ts).

You should also set the value to retryLimit in the constructor function.

Reply:
> We shouldn't read .env variable here. It should be read [here](https://github.com/devpanther/bounty-bot/blob/logs-to-supabase/src/bindings/config.ts).
> 
> You should also set the value to retryLimit in the constructor function.

Done

Reply:
@devpanther If you can link the fix commit/PR before you mark the conversation resolved so it would make reviewers easy to confirm.